Seminar1/Task7.py

```diff
@@ -0,0 +1,36 @@
+# Задание №7
+# Пользователь вводит число от 1 до 999. Используя операции с числами
+# сообщите что введено: цифра, двузначное число или трёхзначное число.
+# Для цифры верните её квадрат, например 5 - 25
+# Для двузначного числа произведение цифр, например 30 - 0
+# Для трёхзначного числа его зеркальное отображение, например 520 - 25
+# Если число не из диапазона, запросите новое число
+# Откажитесь от магических чисел
+# В коде должны быть один input и один print
+
+
+numberType = ""
+numberCounter0 = 10
+numberCounter00 = 100
+result = 0
+min = 0
+max = 999
+while True:
+    number = int(input("введите число от 1 до 999: "))
+    if (min < number | number < max):
```

Comment:
Во-первых, скобки не нужны. Во вторых, не используйте битовые операторы, они работают медленней. В третьих тут должен быть оператор `&` (`and`). Но если тут оператор `and`, то без него вообще можно обойтись. В четвертых, при вашем условиях 0 и 999 учитываться не будут
```suggestion
    if min <= number <= max:
```